index.js

```diff
@@ -270,11 +270,11 @@ class Peer extends stream.Duplex {
    * Add a MediaStream to the connection.
    * @param {MediaStream} stream
    */
-  addStream (stream) {
+  addStream (stream, triggerNegotiate = true) {
```

Comment:
I don't actually want to add this to the public API, but this PR is just to demonstrate a fix to the problem